integration-notification-service/src/main/java/md/maib/integration/notification/service/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,7 @@
+package md.maib.integration.notification.service.config;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RabbitMQConfig {
+}
```

Comment:
why it is a good practice to end a file with empty line?
Version Control Clarity: When you add a line without an ending newline, some version control systems (like Git) may mark the file with a warning about a missing newline, or display differences strangely. Having a newline ensures more consistent and clean diffs, which helps during code reviews and when merging changes.